jakartaee/src/main/java/com/azure/samples/todo/ToDoResource.java

```diff
@@ -36,7 +36,6 @@ public ToDoItem create(@Valid ToDoItem item) {
     public void edit(
             @PathParam("id") Long id,
             @Valid ToDoItem item) {
-        item.setId(id);
```

Comment:
Please do this for all the versions of this code and also please remove the path parameter.

Reply:
Done.